lib/components/narrative/tabbed-itineraries.js

```diff
@@ -64,49 +66,57 @@ class TabbedItineraries extends Component {
         !useRealtime
       )
     return (
-      <div className='options itinerary tabbed-itineraries'>
-        <div className='tab-row'>
-          {itineraries.map((itinerary, index) => {
-            return (
-              <TabButton
-                currency={currency}
-                defaultFareKey={defaultFareKey}
-                index={index}
-                isActive={index === activeItinerary}
-                itinerary={itinerary}
-                key={index}
-                onClick={setActiveItinerary}
-              />
-            )
-          })}
-        </div>
+      <>
+        {error && itineraries && <div className='error-message'>
+          <div className='header'>
+            <i className='fa fa-exclamation-circle' /> Warning
+          </div>
+          <div className='message'>{error.msg}</div>
```

Comment:
Display the localized error message based on error code, then fallback on the error message. See https://github.com/opentripplanner/otp-react-redux/blob/05db79c6e1852f8cab72467fd8bbac94c9198304/lib/components/form/error-message.js#L11.

Reply:
Or use the `<ErrorMessage>` component here instead!

Reply:
Good find thank you! Much much cleaner in ae51c07e1e6bbf86d5e1edd1c32408e0ef025295